cmd/istioctl/main.go

```diff
@@ -58,6 +64,20 @@ var (
 		Long: fmt.Sprintf("Istio configuration command line utility. Available configuration types: %v",
 			model.IstioConfig.Kinds()),
 		PersistentPreRunE: func(*cobra.Command, []string) (err error) {
+			// Get manager address
+			add := os.Getenv("ISTIO_MANAGER_ADDRESS")
+			if add == "" {
+				return errors.New("manager address environment variable is not set," +
```

Comment:
can you add a flag and populate it with env variable value, just like other flags do?